test/integration/tests/post_Invite_User.ts

```diff
@@ -0,0 +1,28 @@
+import {Error} from 'tslint/lib/error';
+import { generatePOSTAPIRequest } from './utils';
+const should = require('chai').should();
+
+suite('API/CASES3 -> POST Invite User', function() {
+  this.timeout(50000);
+
+  const payload = {
+    firstName: 'Vamshi',
+    lastName: 'Muniganti',
+    email: 'vam.mun1752@mailnesia.com',
+    roles: ['pui-organisation-manager'],
+    resendInvite: true,
+  };
+  test('POST Invite User', () => generatePOSTAPIRequest ('POST', '/api/reinviteUser?organisationId=2GIHJH9', payload)
+     // console.log('response', response.headers.get('cache-control'))
+        .then(response => {
+          if (response.status === 429 ) {
+          console.log(`User Already Invited: ${response.status}`);
```

Comment:
missing assertions statements. please add expect or assert statements .

Reply:
dont need assertion as its tricky to validate the email id which activates every hour, so basically verifying the response codes and it throws any exception in fetch